jobs/refresh-hiv-data.yml

```diff
@@ -1,15 +1,13 @@
 # For the HIV Server, we only execute the "hiv", "ovc", and "hivmigration" ETL jobs
-
 type: "job-pipeline"
 description: "Refresh HIV Data for ${siteName}"
 parameters:
   siteName: "hiv"
-  partitionNum: "8"
+  partitionNum: "1"
 schedule:
   cron: "${executeCron.refreshHivData}"
 configuration:
   jobs:
-    - path: "create-source-views-and-functions.yml"
```

Comment:
You should keep this here, and get rid of the "refresh-functions-n-views.yml" job you created and added below.  The reason is that we only need/want to create views and functions once per source DB, not for every single table that we are importing.